packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -432,6 +432,9 @@ export function createAPIFactory(
             createInputBox(): theia.InputBox {
                 return quickOpenExt.createInputBox(plugin);
             },
+            registerTerminalLinkProvider(provider: theia.TerminalLinkProvider): void {
+                /* NOOP */
```

Comment:
maybe the comment should say that it is an empty stub for now and to be implmented in a later stage